vehicles/src/common/helper/permit-application.helper.ts

```diff
@@ -10,6 +10,14 @@ import { callDatabaseSequence } from './database.helper';
 import { PermitApplicationOrigin as PermitApplicationOriginEnum } from '../enum/permit-application-origin.enum';
 import { PermitApprovalSource as PermitApprovalSourceEnum } from '../enum/permit-approval-source.enum';
 import { randomInt } from 'crypto';
+import { Directory } from '../enum/directory.enum';
+import { doesUserHaveAuthGroup } from './auth.helper';
+import {
+  IDIR_USER_AUTH_GROUP_LIST,
+  UserAuthGroup,
+} from '../enum/user-auth-group.enum';
+import { PPC_FULL_TEXT } from '../constants/api.constant';
+import { User } from '../../modules/company-user-management/users/entities/user.entity';
```

Comment:
@praju-aot Will this import cause circular dependency?

Reply:
No, it wont